src/styles/core/_variables.scss

```diff
@@ -203,27 +203,29 @@ $dc-z-index100: 9999 !default;
 
 // Base is 16px at this point
 
-// Feature phone: 0px
-$dc-tiny-width: 0 !default;
+/// This bloats the CSS use only those which you need
+///
```

Comment:
@frontendherodk Yes, same as Giant and Monstrous width, this should be added back. Is part of my future commit.